packages/cli/src/utils/platform.ts

```diff
@@ -59,6 +59,21 @@ export const getRequiredParameter = <$Args extends Record<string, unknown>, $Nam
   return value
 }
 
+export const ErrorPlatformUnauthorized = new Error(
+  'No platform credentials found. Please provide a token via --token or -t, or add PRISMA_TOKEN environment variable or run `prisma platform login`.',
```

Comment:
```suggestion
  'No platform credentials found. Run `prisma platform login` first. Alternatively you can provide a token via the `--token` or `-t` parameters, or set the `PRISMA_TOKEN` environment variable with the token.',
```

I would first suggest login.

Reply:
You could make it look like other instructions in other parts in the CLI
Like 

```
`Run ${green(getCommandWithExecutor('prisma platform login'))}`
```

`getCommandWithExecutor` is from `@prisma/internals`

Reply:
Amazing, thanks @Jolg42! I will update the copy and highlight the command

Reply:
@Jolg42 did you intentionally use back ticks  around parameters or are single quotes ok? 

Example:  `'--token'`

Reply:
In general, we use backticks for code quotes, so for consistency